.github/workflows/ci.yaml

```diff
@@ -147,3 +146,45 @@ jobs:
       - name: Run tests
         working-directory: ./ui
         run: npm run test
+
+  # This job builds the Docker images for the controller, UI, app, and CLI on arm64.
+  build-multiarch:
```

Comment:
Can you change this name back to `build`. All of the PR requirements expect a job named `build`